app/rpc.js

```diff
@@ -37,6 +37,18 @@
         "thread-open": async function(threadId) {
             await F.mainView.openThreadById(threadId);
         },
+         
+        "thread-call-start": async function(threadId, autoJoinBool) {
+            const thread = requireThread(threadId);
+            const callMgr = F.calling.getOrCreateManager(thread.id, thread);
+            await callMgr.start({autoJoin: autoJoinBool});
+        },
+ 
+        "thread-call-join": async function(threadId) {
+            const thread = requireThread(threadId);
+            const callMgr = F.calling.getOrCreateManager(thread.id, thread);
+            await callMgr.start({autoJoin: true});
+        },
```

Comment:
Let's rework this one a bit.  Instead of pigging backing on the autoJoin behavior or a call managers start method, I think it's better if we just call the join method of an existing call.  something like..

```javascript
        "thread-call-join": async function(callId) {
￼           const callMgr = F.calling.getManager(callId);
            if (!callMgr) {
                throw new ReferenceError("Invalid callId");
            }
            if (!callMgr.view) {
                throw new TypeError("Call does not have bound view");
            }
￼           await callMgr.view.join();
￼        },
```